PlaybackLab/TVOSSwiftSampleApp/TVOSSwiftSampleApp/Players/ChildPlayerViewController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  ChildPlayerViewController.swift
+//  TVOSSwiftSampleApp
+//
+//  Copyright © 2018 Ooyala. All rights reserved.
+//
+// This example shows shows a use case in which you want to show the player next to
+// other UI in the screen, so the player will not be in fullscreen mode taking all
+// your screen space, but it will be sharing the UI with other elements which could
+// be actionable too, like buttons.
+//
+// This is not a verified use case by Ooyala and cause extrange behavior for the player,
+// but we provide the example in case you want to use it.
+
+import Foundation
+
+class ChildPlayerViewController: AbstractPlayerViewController {
+  
+  @IBOutlet var playerView: UIView!
+  var pcode: String!
```

Comment:
Why we need to store a _pcode_ as a variable if we have `option` in the parent class?

Reply:
I removed it